Taskfile.yaml

```
@@ -0,0 +1,192 @@
+      - task: lint
+      - task: test
+
+  encrypt:
+    desc: "Encrypt .env files"
+    cmds:
+      - bash ./scripts/encrypt.sh
+
+  decrypt:
+    desc: "Decrypt .env files"
+    cmds:
+      - bash ./scripts/decrypt.sh
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)